web/src/components/molecules/Common/projectMenu/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {
```

Comment:
This and the workspaceMenu are VERY similar (identical?). Can you create a Common/MenuList component (if there is a better name, please use that instead)?

Reply:
Yup the reason it's not like that right now it is done on a branch of the schema UI, will create that PR later.